### ISSUE TITLE
Bad layout of settings dialog in Android

### ISSUE THREAD

**Issue by j-santander** [jamulussoftware/jamulus#1529]
**Describe the bug**

With the code from master, the settings dialog in android is not layout correctly, with the rows appearing *compressed* as the screen is not tall enough.

**To Reproduce**

Install the latest application in android.

**Expected behavior**

Dialog should be layed out properly.

**Screenshots**

![Screenshot_20210417-191524](https://user-images.githubusercontent.com/2654578/115121179-90111b00-9fb1-11eb-99fa-20afacedb4b8.png)

**Operating system**

Android 11 - Google Pixel 3a

**Version of Jamulus**

Built from source at 3df60d6f9faf9bd9fc421187f4e62650abf0b1e0

**Additional context**

**Comment by ngocdh** [comment]
I confirm the issue. And it’s actually worse for iOS. In iOS, the issue exists for ClientDlg as well, where Android does much better.
This might have to do with how Qt deals with different OSes.

**Comment by DavidSavinkoff** [comment]
You can get 20% more room! (3 FAT lines worth) without resorting to scroll-bars.
- put Device beside the selector (as Audio Channels and Audio Quality is)
- do the same with Buffer Delay (Buffer is good enough Delay is too much)
- get rid of Measurements (heading)

**Comment by ngocdh** [comment]
> I confirm the issue. And it’s actually worse for iOS. In iOS, the issue exists for ClientDlg as well, where Android does much better.
> This might have to do with how Qt deals with different OSes.

This is how it looks actually in iOS.

![47A89825-4655-4124-B4C9-8C5D5C926B44](https://user-images.githubusercontent.com/6765612/115148058-33b80500-a05e-11eb-9030-ef8b8150d809.png)
![C009C8F1-32C2-4415-90DE-11C9872C62FB](https://user-images.githubusercontent.com/6765612/115148061-3581c880-a05e-11eb-95fe-4217ef894b11.png)

**Comment by ngocdh** [comment]
Could anyone point me in the right direction to fix the UI for iOS? I couldn't figure it out, since the same code is working quite well for other platforms, even Android. Somehow the ClientDlg now looks even worse: I can now see only 2 sliders (the lower part only) in the mixer.

**Comment by ann0see** [comment]
You could open the UI in Qt Designer?

**Comment by ngocdh** [comment]
> You could open the UI in Qt Designer?

Yes.

**Comment by ann0see** [comment]
Ok. Probably you need to play CX around with the spacers/items there to get a working UI

**Comment by ngocdh** [comment]
I tried. But couldn't make it work due to my very limited experience. There are two problems:

- For iOS: it seems the UI is rendered in low resolution, in contrary to Android, where everything is rendered perfectly
- For touch screens in general: The mixer should support touch screen so that user can navigate easily, small scrollbars around the mixer is not the best solution here. Qt must surely support touch screen right?

**Comment by ann0see** [comment]
> Qt must surely support touch screen right?

Yes but this requires QML and probably a large rewrite of the whole UI

**Comment by pljones** [comment]
I think we probably do need a complete UI rewrite in the near future for mobile support.
![Screenshot_20230812-121222](https://github.com/jamulussoftware/jamulus/assets/1549463/1558e173-38e8-4cce-93bd-2d674a00f72c)
Decisions about how to lay out the screen haven't been based on which controls are _necessary for use_ and which are merely informational.

**Comment by ann0see** [comment]
Agree. There were concepts for a new UI somewhere.
